anvil_consortium_manager/templates/anvil_consortium_manager/workspace_landing_page.html

```diff
@@ -0,0 +1,52 @@
+{% extends "anvil_consortium_manager/base.html" %}
+{% load static %}
+
+{% block title %}Workspaces{% endblock %}
+
+{% block content %}
+
+
+<h1 class="pb-2 border-bottom">Workspaces</h1>
+
+
+<div class="album py-2">
+  <div class="container">
+    <div class="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-3">
+
+      {% for workspace_type, workspace_name in registered_workspaces.items %}
+      <div class="col">
+        <div class="card shadow-sm">
+          <h5 class="card-header"><span class="fa-solid fa-computer mx-2"></span>{{ workspace_name }}s</h5>
+          <div class="card-body">
+            <p>
+              Interact with {{ workspace_name }}s on AnVIL.
+              {% if perms.anvil_consortium_manager.anvil_project_manager_edit %}You can create or import workspaces and then share them with a group.{% endif%}
```

Comment:
I'm planning to change this by adding a required description for the workspace adapters. Then we can have that description appear in the cards on the landing page.

I'll make these specific changes once you're done with your part of the branch, or I'll do it in a new branch.